src/activations.jl

```diff
@@ -33,13 +33,13 @@ end
 const sigmoid = σ
 
 """
-    hardσ(x, a=0.2) = max(0, min(1, a * x + 0.5))
+    hardσ(x) = max(0, min(1, (x + 1)/2)
 
 Segment-wise linear approximation of sigmoid.
 See [BinaryConnect: Training Deep Neural Networks withbinary weights during propagations](https://arxiv.org/abs/1511.00363).
 """
-hardσ(x, a=0.2) = oftype(x/1, max(zero(x/1), min(one(x/1), oftype(x/1,a) * x + oftype(x/1,0.5))))
-    
+hardσ(x) = max(zero(x), min(one(x), (x + 1) / 2))
```

Comment:
the default value of `a` was in disagreement with the one in the paper. Also, the paper doesn't mention any tunable parameter `a`